nvi-test/src/main/java/no/sikt/nva/nvi/test/ExpandedContributor.java

```diff
@@ -0,0 +1,125 @@
+package no.sikt.nva.nvi.test;
+
+import static java.util.Objects.nonNull;
+import static no.sikt.nva.nvi.test.TestConstants.CONTRIBUTOR;
+import static no.sikt.nva.nvi.test.TestConstants.CREATOR;
+import static no.sikt.nva.nvi.test.TestConstants.IDENTITY;
+import static no.sikt.nva.nvi.test.TestConstants.IDENTITY_FIELD;
+import static no.sikt.nva.nvi.test.TestConstants.ID_FIELD;
+import static no.sikt.nva.nvi.test.TestConstants.NAME_FIELD;
+import static no.sikt.nva.nvi.test.TestConstants.ROLE_FIELD;
+import static no.sikt.nva.nvi.test.TestConstants.TYPE_FIELD;
+import static no.unit.nva.testutils.RandomDataGenerator.objectMapper;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.net.URI;
+import java.util.List;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+public record ExpandedContributor(URI id, String verificationStatus, String contributorName, String role,
+                                  List<ExpandedAffiliation> affiliations, String orcId) {
+
+
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static Builder from(ExpandedContributor other) {
+        return new Builder(other);
+    }
+
+    public ObjectNode asObjectNode() {
+        var contributorNode = objectMapper.createObjectNode();
+        contributorNode.put(TYPE_FIELD, CONTRIBUTOR);
+
+        contributorNode.set("affiliations", createAndPopulateAffiliationsNode());
```

Comment:
"affiliations" is used in classes as ExpandedContributor, Contributor, NviContributor, ExpandedResourceGenerator and more. Maybe as constant?

Reply:
Agreed, but we can do more of this in a separate PR. Refactored some of it now.